### DISCUSSION TITLE
HTML redirection - There isn't a GitHub Pages site here.

### DISCUSSION THREAD

**Discussion by florencioMath** [community/community#35955] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I'm trying to make an index.html lead to another index.html that is inside a page, I can without problems with the Live Server but when I try with GitHub pages it gives the error 404 - There isn't a GitHub Pages site here.

It goes from here.
![image](https://user-images.githubusercontent.com/68713424/195445609-fe0b55c5-c422-4b93-a70d-d720b8eae97b.png)

to here:
![image](https://user-images.githubusercontent.com/68713424/195445710-c9e85440-f58f-4d10-986b-a860aa353567.png)
---

But when I change the adress from: _https://florenciomath.github.io/expanding-cards/index.html_ to _https://florenciomath.github.io/Project-JavaScript/expanding-cards/index.html_ works!

![image](https://user-images.githubusercontent.com/68713424/195446859-9c3d34c0-c4a0-4c3d-9532-0abb9017cc24.png)
---

Here is the code:
![image](https://user-images.githubusercontent.com/68713424/195446218-2ff05b1d-3409-42ab-9c33-34116af8d40e.png)
---

I tried putting all the files inside a folder "/Project-JavaScript/" but it didn't work, could someone help me on how I can go from index.html from the root of the project to an index inside a folder using Github Pages?

Repository: https://github.com/florencioMath/Project-JavaScript
Site: https://florenciomath.github.io/Project-JavaScript/

**Comment by radar-blip** [comment]
So, just change the link in your index.html from:
`<a href="../expanding-cards/index.html">Expanding Cards</a>`
To:
`<a href="Project-JavaScript/expanding-cards/index.html">Expanding Cards</a>`


Will that work like you want?

**Reply by florencioMath** [reply]
Hi [linuxlawson](https://github.com/linuxlawson)!

Thank you, for the replay!

It will solve the problem, I take that like a last resort, because when I'm coding and using Live Server it will take me to the GitHub pages and not the development enviroment, I could change the href when I make the final commit but I'm trying to find another solution for this.

**Reply by radar-blip** [reply]
It seems I just repeated everything you already tried, sorry bout that, not enough coffee today!
Anyway, I'm not familar with how Live Server (vscode?) works, so im afraid I wont be of much help here.
Keep Trying!

**Comment by florencioMath** [comment, marked as answer]
I think it resolved itself. 🤔 

Today the GitHub pages is putting the "/Project-JavaScript/" on the route and the page is working fine. Strange hum?!
And I even created a function to add the "/Project-JavaScript/" when the route contains "github.io", well I got experience from that! hahahah

Thanks for the help!
